ftag/vds.py

```diff
@@ -58,6 +58,9 @@ def create_virtual_file(
         for group in h5py.File(fnames[0]):
             layout = get_virtual_layout(fnames, group)
             f.create_virtual_dataset(group, layout)
+            with h5py.File(fnames[0]) as g:
+                 for name, value in g[group].attrs.items():
+                      f[group].attrs[name] = value
```

Comment:
I think we should probably consider the case where the different files have different/conflicting attributes.
One way to do this would be to loop over all input files and add a dict of list
```python
for name, value in g[group].attrs.items():
    if name not in attrs_dict:
        attrs_dict[name] = []
    attrs_dict[name].append(value)
```
we can then check which entries have `len(set(attrs_dict[name]))==1` and add only these to the output file (possibly raising a warning about other attributes)

Reply:
Hi @longwills, are you able to follow up on this comment? thanks!